app/i18n/fr/sub.php

```diff
@@ -1,6 +1,10 @@
 <?php
 
 return array(
+	'api' => array(
+		'documentation' => 'Copier l\'URL suivante pour l\'utiliser dans l\'outil qui consommera l\'API.',
```

Comment:
See, even in French you're using use, although apparently the tool, consumes the API. :-P

Reply:
In French, the consume translation seems odd. That's why I used "use".

Reply:
Had I read it in French I would have thought "oh well, I guess that's French", or perhaps more Obélix-style "ils sont fous, ces français" but I can confidently say that I don't much like it in English. ;-)

Reply:
May I suggest: `'Copier l’URL suivante dans l’outil qui utilisera l’API.'` ?
And I would rather go with *Unicode curly apostrophe* rather than *backslash straight single quote*.